pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -221,6 +245,29 @@ function Meta(meta)
   end
 
   zotero.url = zotero.url .. '&citationKeys='
+
+  if string.match(FORMAT, 'odt') and config.csl_style ~= '' then
+    meta.ZOTERO_PREF_1 = string.format(
+      '<data data-version="3" zotero-version="5.0.89">'
+        .. '   <session id="OGe1IYVe"/>'
+        .. '   <style id="http://www.zotero.org/styles/%s" locale="en-US" hasBibliography="1" bibliographyStyleHasBeenSet="0"/>'
+        .. '   <prefs>'
+        .. '     <pref name="fieldType" value="ReferenceMark"/>'
+      -- .. '     <pref name="delayCitationUpdates" value="true"/>'
+        .. '   </prefs>'
+        .. '</data>',
+      config.csl_style)
+    meta.ZOTERO_PREF_2 = ''
+  end
+
+  print('')
```

Comment:
I'd rather not print stuff by default. If we keep this, then as an option.

Reply:
> I'd rather not print stuff by default. If we keep this, then as an option.

I thought you might want to unit test my changes.  Do you want this portion removed, or commented.